tests/cases/fourslash/getOccurrencesAfterEdit.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path='fourslash.ts'/>
+
+/////*0*/
+////function f(s: string) {
+////    s.con/*1*/structor

```

Comment:
What's s.constructor? It seems like a weird thing to be finding references to. Can you add a test where the identifier has other references besides the starting location?